contracts/v1/FiatTokenV1.sol

```diff
@@ -119,7 +119,7 @@ contract FiatTokenV1 is AbstractFiatTokenV1, Ownable, Pausable, Blacklistable {
      * @return True if the operation was successful.
      */
     function mint(address _to, uint256 _amount)
-        external
+        public
```

Comment:
I am not quite sure we want to change this, especially if we want to get this merged back into upstream.

We can keep mint `external`, but instead call it by using `FiatTokenV1(address(this)).mint(...)` from the token. This would require granting the token minter role but should be ok.

Reply:
Set back to external. Due to token address being blacklisted in one of the initializers, burn/mint logic is still duplicated in internal functions. Contract size reduced by lowering optimizer runs